sqlmesh/core/config/ui.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import typing as t
+
+from pydantic import Field
+
+from sqlmesh.core.config.base import BaseConfig
+
+
+class FormatOptions(BaseConfig):
```

Comment:
i don't think this belongs in ui.py, since this is also used for the cli isn't it

Reply:
Done